### PR TITLE
reverse sync

### PR DESCRIPTION
**Pull request by psi29a**
I think it is a good idea to keep the two repos in sync.

This covers:
1) CI fixes
2) Being explicit about the license 

That way in the future, that there is no more merge conflicts when syncing. :)

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -38,5 +38,21 @@
             android:configChanges="keyboardHidden|orientation"
             android:screenOrientation="sensorLandscape"
             android:theme="@style/Theme.AppCompat.FullScreen" />
+
+        <activity
```
Comment (terabyte25) [inline app/src/main/AndroidManifest.xml]: We need to remove the activity part, but not sure about the fabric apikey though. For me I need the apikey to compile.

File: .travis.yml [hunk 2]
```diff
@@ -3,12 +3,15 @@ sudo: false
...
 
 android:
   components:
     - tools
-    - build-tools-26.0.2
+    - build-tools-28.0.3
     - platform-tools
     - tools
-    - android-27
+    - android-28
```
Comment (terabyte25) [inline .travis.yml]: Is it okay for us to update to version 28 of the API?

Comment (psi29a) [reply .travis.yml]: You're the one who bumped it. You should know. :)

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (psi29a) [reply app/src/main/AndroidManifest.xml]: Can this stay for now until the two forks are in sync, then go about cleaning up on xyzz's fork? I want to be able to do a clean sync back upstream.

Comment (terabyte25) [reply app/src/main/AndroidManifest.xml]: Sounds good to me.

File: .travis.yml (hunk 2 above)
Comment (terabyte25) [reply .travis.yml]: Not exactly, as I believe xyzz used a build tool outside of Android API 27.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -38,5 +38,21 @@
...
+            android:label="Choose File"
+            android:screenOrientation="sensorLandscape">
+
+            <intent-filter>
+                <action android:name="com.libopenmw.openmw.FileChooser" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </activity>
+        
+        <meta-data
```
Comment (xyzz) [inline app/src/main/AndroidManifest.xml]: The apikey should be removed, it was a mistake/laziness on my part.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (xyzz) [inline app/src/main/AndroidManifest.xml]: This activity doesn't exist so should be removed.

File: app/build.gradle [hunk 4]
```diff
@@ -14,13 +14,18 @@ apply plugin: 'kotlin-android'
 ext {
     versionFile = new File(project.rootDir, 'app/src/main/assets/libopenmw/resources/version')
     calculateVersion = {
-        stream = new FileInputStream(versionFile)
-        return stream.readLines()[0]
+        if (versionFile.exists()) {
```
Comment (xyzz) [inline app/build.gradle]: Why's that needed?

Comment (terabyte25) [reply app/build.gradle]: https://travis-ci.org/OpenMW/android-port/builds/524185792#L1566

Comment (psi29a) [reply app/build.gradle]: Was a fix from @terabyte so that travis-ci could build.

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (psi29a) [reply app/src/main/AndroidManifest.xml]: will this break CI?

Comment (xyzz) [reply app/src/main/AndroidManifest.xml]: I don't know, I don't use travis for this repo. I was actually going to remove everything travis-related, but if it makes your life easier I can keep it around.

Comment (terabyte25) [reply app/src/main/AndroidManifest.xml]: For me removing the apikey broke the CI. You can see this here: https://travis-ci.org/OpenMW/android-port/builds/524194894#L1569

**Comment by terabyte25** [comment]
@xyzz can't you just squash this whole PR when you merge it? No need to remove the merge commits.

**Comment by xyzz** [comment]
I can, but
1) it makes it more difficult to review
2) I'm mostly asking to confirm squashing it wouldn't result in any unexpected breakage on your side

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (xyzz) [reply app/src/main/AndroidManifest.xml]: In this case you should figure how to fix the CI (maybe again with a build flavor). Keeping a fabric key hardcoded isn't a good idea especially for an open source project (consider forks/different package names).

**Comment by psi29a** [comment]
Squashing this will cause the two forks to diverge again... which is what we are trying to avoid.

My idea was to sync both and either do the following:
1) xyzz develops on openmw/android-port (taking advantage of OpenMW's CI infra)
2) xyzz developers on xyzz/openmw-android, and we sync back to openmw/android-port, effectively making xyzz/openmw-android default/defacto development fork.

**Comment by xyzz** [comment]
Well, I can either merge a reasonably squashed version of this or not at all. If you want to track this repo, then just force-push it or if you want to track it and have some changes, include merge commits on your side or constantly rebase (and force-push) it, just like I'm doing with https://github.com/xyzz/openmw/commits/android

**Comment by psi29a** [comment]
@terabyte25 perhaps you should fix this:
https://4pda.ru/forum/index.php?showtopic=889357&st=4600#entry83782766

> TES3MP для Android - это форк OpenMW от terabyte25 реализующий многопользовательский функционал.

You should be clearer in your language and use of terms.  TES3MP Android port by terabyte25 - TES3MP is a multiplayer fork of OpenMW.

It clarifies that it is a port of an existing project, by you, for Android. It also clarifies what TES3MP is to OpenMW, which is a multiplayer fork. 

Changing the wording can make things seem more legit and clear up boundary issues.

@xyzz if I track your repo as-is, then CI is broken. What do you suggest that would allow me to track your repo without breaking CI? :)  Another option would be to develop directly on OpenMW/android-port (I can rename if it makes a difference) and you become OpenMW's official Android champion.

**Comment by terabyte25** [comment]
@psi29a Yes, I will make these changes once the curator of said topic returns. And also, I did not write said caption, this was the work of the curator, who i will contact to change this (but this may take a while since the curator is MIA and even if I had edit abilities to post, I know little to no Russian to make any sensible sentences.)

**Comment by terabyte25** [comment]
Should be fixed now: https://4pda.ru/forum/index.php?showtopic=889357&st=4600#entry83782766

**Pull request closed without merging**